about.html

```diff
@@ -0,0 +1,155 @@
+        </div>
+        <div class="about-section__containers">
+          <img class="about-section__logo" src="./images/icons/mainIcon.png" alt="best games logo">
+        </div>
+      </div>
+      <div class="about-section__titles">
+        <div class="about-section__containers">
+          <h2 class="about-section__main-title">See the best rated games ever</h2>
+          <div class="orange-line"></div>
+        </div>
+        <div class="about-section__containers">
+          <p class="about-section__description">Take a look at the two best seller games not only in Best games! but worldwide</p>
+        </div>
+      </div>
+      <div class="about-section__images">
+        <div class="about-section__image-container container-solo">
+          <div class="about-section__orange">
+            <p class="about-section__title-one">Until Dawn</p>
+            <p class="about-section__title-two">Best rated game</p>
+          </div>
+          <img class="about-section__image" src="./images/skullBackground.jpg" alt="until down image">
+        </div>
+        <div class="about-section__image-container">
+          <div class="about-section__orange">
+            <p class="about-section__title-one">Skyrim</p>
+            <p class="about-section__title-two">Second best rated game</p>
+          </div>
+          <img class="about-section__image" src="./images/skyrimBAckground.jpg" alt="skyrim image">
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <!-- Footer section -->
+
+  <footer class="footer-section">
+    <div class="footer-section__container">
+      <div class="footer-section__first-part">
+        <div class="footer-section__title-container">
+          <p class="footer-section__title">Partner</p>
+          <div class="orange-line"></div>
+        </div>
+        <div class="footer-section__list-container">
+          <ul class="footer-section__first-logos">
+            <li><img class="footer-section__image" src="./images/mozillaL.png" alt="mozilla logo"></li>
+            <li><img class="footer-section__image" src="./images/pngfind.com-flat-google-logo-png-4206174.png" alt="google logo"></li>
+            <li><img class="footer-section__image naver" src="./images/naverLogo.png" alt="naver logo"></li>
+          </ul>
+          <ul class="footer-section__last-logos">
+            <li><img class="footer-section__image kakao" src="./images/kakao.png" alt="kakao logo"></li>
+            <li><img class="footer-section__image" src="./images/airBnbLogo.png" alt="air logo"></li>
+          </ul>
+        </div>
+      </div>
+      <div class="footer-section__second-part">
+        <div class="footer-section__logo-container"><img class="footer-section__logo" src="./images/icons/mainIcon.png" alt="best games icon"></div>
+        <div class="footer-section_text-container"><p class="footer-section__small-text">Buy games!<br>Some Rights Reserved</p></div>
+      </div>
+    </div>
+  </footer>
```

Comment:
- [ ] I see that you made an effort in centering the content of the footer, but the `BG` logo is still on the far left. Kindly move it to the center with the text as well. See the image below for a guide. 


![image](https://github.com/EvansSnave/capstone-project/assets/84156714/78017db1-28ad-4c72-b9dd-938814e6e492)